plugins/ecs-bridge/engine/engine.go

```diff
@@ -190,8 +190,7 @@ func (engine *engine) RunIPAMPluginAdd(plugin string, netConf []byte) (*current.
 
 	result, err := current.NewResultFromResult(ipamResult)
 	if err != nil {
-		return nil, errors.Wrapf(err,
-			"bridge IPAM ADD: unable to parse result '%s'", ipamResult.String())
+		return nil, errors.Wrapf(err, "bridge IPAM ADD: unable to parse result.")
```

Comment:
Why are we making the change? I'd imagine that printing `ipamResult` could be helpful in understanding why we failed to parse?

Reply:
updated and added test.